ui/components/app/edit-gas-fee-popover/network-statistics/status-slider/status-slider.test.js

```diff
@@ -1,51 +1,81 @@
 import React from 'react';
-import { screen } from '@testing-library/react';
 
 import { renderWithProvider } from '../../../../../../test/jest';
-import { ETH } from '../../../../../helpers/constants/common';
-import { GasFeeContextProvider } from '../../../../../contexts/gasFee';
+import { GasFeeContext } from '../../../../../contexts/gasFee';
 import configureStore from '../../../../../store/store';
 
 import StatusSlider from './status-slider';
 
-jest.mock('../../../../../store/actions', () => ({
-  disconnectGasFeeEstimatePoller: jest.fn(),
-  getGasFeeEstimatesAndStartPolling: jest
-    .fn()
-    .mockImplementation(() => Promise.resolve()),
-  addPollingTokenToAppState: jest.fn(),
-  getGasFeeTimeEstimate: jest
-    .fn()
-    .mockImplementation(() => Promise.resolve('unknown')),
-}));
-
-const renderComponent = () => {
-  const store = configureStore({
-    metamask: {
-      nativeCurrency: ETH,
-      provider: {},
-      cachedBalances: {},
-      accounts: {
-        '0xAddress': {
-          address: '0xAddress',
-          balance: '0x176e5b6f173ebe66',
-        },
-      },
-      selectedAddress: '0xAddress',
-    },
-  });
-
-  return renderWithProvider(
-    <GasFeeContextProvider>
+const renderComponent = ({ networkCongestion }) => {
+  const component = (
+    <GasFeeContext.Provider value={{ gasFeeEstimates: { networkCongestion } }}>
```

Comment:
The component doesn't really care how the context is created, just that it is there, so to me it was simpler to use GasFeeContext.Provider than GasFeeContextProvider.

Reply:
Looks good 👍